### PR TITLE
tightenging tolerance for MS cases for regression test

### PR DESCRIPTION
**Pull request by GitPaean**

### CODE REVIEW

**Comment by GitPaean** [comment]
jenkins build this update_data please

**Comment by GitPaean** [comment]
With shortening time step to be five days, two of the failing tests recovered and the following one remains to be fixed. 
```
The following tests FAILED:
	 57 - compareECLFiles_flow+TEST_WSEGSICD (Failed)
Errors while running CTest
```

File: compareECLFiles.cmake [hunk 1]
```diff
@@ -298,7 +299,8 @@ add_test_compareECLFiles(CASENAME msw_model_1
                          SIMULATOR flow
                          ABS_TOL ${abs_tol}
                          REL_TOL ${rel_tol}
-                         DIR model1)
+                         DIR model1
+                         TEST_ARGS --tolerance-pressure-ms-wells=10. --tolerance-wells=1.e-6 --solver-max-time-step-in-days=5.0)
```
Comment (alfbr) [inline compareECLFiles.cmake]: Great to see this one pass. Have you checked that the first two contstraints are still needed, i.e., could it be that max time step is sufficient alone?

File: compareECLFiles.cmake [hunk 2]
```diff
@@ -312,7 +314,8 @@ add_test_compareECLFiles(CASENAME faults_model_1
...
```
Comment (alfbr) [inline compareECLFiles.cmake]: Same here, great to see this one recovered, but are we sure we need more that just the time step constraint?

File: compareECLFiles.cmake [hunk 3]
```diff
@@ -395,7 +398,8 @@ add_test_compareECLFiles(CASENAME wsegsicd
 			  FILENAME TEST_WSEGSICD
 			  SIMULATOR flow
 			  ABS_TOL ${abs_tol}
-			  REL_TOL ${rel_tol})
+			  REL_TOL ${rel_tol}
+              TEST_ARGS --tolerance-pressure-ms-wells=10. --tolerance-wells=1.e-6 --solver-max-time-step-in-days=5.0)
```
Comment (alfbr) [inline compareECLFiles.cmake]: This one seems tricky, I suggest we go back to the drawing board on it, and leave it out of the pull request.

**Comment by alfbr** [comment]
I am happy to see this merged, once we have made sure that we are not using more constraints than needed.

**Comment by GitPaean** [comment]
> I am happy to see this merged, once we have made sure that we are not using more constraints than needed.

We can try to find the least modification needed.

**Pull request closed without merging**